faucet/open_channel.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
```

Comment:
this comment is meant for above for `faucet/macaroons.db`: did you intent to check this in?

Reply:
yes, as the macaroons.db contains the keys for the macaroon generation. We need to keep them the same so we can pass the admin.macaroon statically in our faucet proxy. Nothing you would consider for production, but more convenient for the tests.